tests/integration/test_pfcoil_int.py

```diff
@@ -271,7 +271,7 @@ def pfcoil():
 #     assert pytest.approx(pfv.rjohc) == -7.728453e9
 
 
-def test_efc(pfcoil: PFCoil[CsFatigue], monkeypatch: pytest.MonkeyPatch):
+def test_efc(pfcoil: PFCoil, monkeypatch: pytest.MonkeyPatch):
```

Comment:
As this corrects an earlier error, can the previous commit be fixed up with this to reduce churn please?

Reply:
Will do using rebase